man/zetta.1.md

```diff
@@ -191,29 +191,29 @@ Directories will be shown to have the status of their contents, which is how ‘
 ENVIRONMENT VARIABLES
 =====================
 
-exa responds to the following environment variables:
+Zetta responds to the following environment variables:
```

Comment:
Current changes look good on my machine.

Maybe to prevent `Issue #XY: Rename the variables to ZETTA_` (for now):
```suggestion
Zetta responds to the following environment variables:
(Note: Compatibility with exa is a priority hence the variable names.)
```